msadgraph.json

```diff
@@ -0,0 +1,2969 @@
+{
+    "appid": "f2a239df-acb2-47d6-861c-726a435cfe76",
+    "name": "MS Graph for Active Directory",
+    "description": "Connects to Microsoft AD using MS Graph REST API services",
+    "type": "identity management",
+    "product_vendor": "Microsoft",
+    "logo": "logo_msadgraph_dark.svg",
+    "logo_dark": "logo_msadgraph_dark.svg",
+    "product_name": "MS Graph for Active Directory",
+    "product_version_regex": ".*",
+    "publisher": "Splunk Community",
+    "contributors": [
+        {
+            "name": "Ekta Bhoir"
+        }
+    ],
+    "license": "Copyright (c) 2019-2022 Splunk Inc.",
```

Comment:
As this is a new release, it should be Copyright (c) 2022 Splunk Inc.

Reply:
updated copyrights